src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -818,16 +821,17 @@ private JPanel getPanelGeneral() {
 			gbc.fill = java.awt.GridBagConstraints.HORIZONTAL;
 			gbc.insets = new java.awt.Insets(2,2,2,2);
 			gbc.anchor = java.awt.GridBagConstraints.WEST;
-            
+

```

Comment:
Formatting changes should be done in a separate PR.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting